Integrations/RTIR/RTIR_description.md

```diff
@@ -0,0 +1,3 @@
+ Additional Parameters:
```

Comment:
lets remove this as well, so it will be reviewed in the next pr

Reply:
Right, missed that...